library/src/main/java/com/google/android/exoplayer/ExoPlayerImplInternal.java

```diff
@@ -500,6 +507,8 @@ private void seekToInternal(long positionMs) throws ExoPlaybackException {
     try {
       if (positionMs == (positionUs / 1000)) {
         // Seek is to the current position. Do nothing.
+        pendingSeek = false;
+        eventHandler.sendEmptyMessage(MSG_SEEK_COMPLETE);
```

Comment:
Note - The seek to the "current position" might not have completed yet, in which case I think this ends up firing the event too soon. For example if you do a seek to t=5000 twice in quick succession:

- The first call will update positionUs at L516 (just below) and pendingSeek will be set to true.
- The second call will arrive almost immediately whilst the player is still buffering, set pendingSeek back to false again and fire the event.

It feels like it would be more correct to remove the two lines being added here, or turn pendingSeek into an integer counter so that you can fire the correct number of MSG_SEEK_COMPLETE events once the seek really has completed.